wellregistry/registry/models.py

```diff
@@ -128,7 +129,7 @@ class Registry(models.Model):
     agency = models.ForeignKey(AgencyLookup, on_delete=models.PROTECT, db_column='agency_cd', null=True,
                                to_field='agency_cd')
     site_no = models.CharField(max_length=16)
-    site_name = models.CharField(max_length=300, blank=True)
+    site_name = models.CharField(max_length=300, validators=[MinLengthValidator(1)])
```

Comment:
There are several fields that can not be blank. This is the change you would make for those.

Reply:
Please also note that we will need to make new migrations even though this doesn't actually change the database. we can talk more on Friday.